pkg/runtime/cli.go

```diff
@@ -342,8 +343,8 @@ func FromFlags() (*DaprRuntime, error) {
 	// Config and resiliency need the operator client
 	var operatorClient operatorV1.OperatorClient
 	if *mode == string(modes.KubernetesMode) {
-		log.Info("Initializing the operator client")
-		client, conn, clientErr := client.GetOperatorClient(*controlPlaneAddress, security.TLSServerName, runtimeConfig.CertChain)
+		log.Infof("Initializing the operator client (config: %s)", *config)
```

Comment:
why's is the whole config printed here? I think that is not needed.

Reply:
We removed in [this commit](https://github.com/dapr/dapr/commit/77136ebd1276d921d7dee5297ced4eab2f59e240#diff-713a1d05d1625fc206303695742169d2c70e5e5d326277965beee9b005b353adL345) on a seperate PR, and git has readded it during rebasing, or GitHub with "merge master".
Will remove.